Doc/whatsnew/3.10.rst

```diff
@@ -82,6 +82,9 @@ New Features
 * :pep:`618`: The :func:`zip` function now has an optional ``strict`` flag, used
   to require that all the iterables have an equal length.
 
+* :pep:`604`: Builtin Union type to allow writing X | Y in place of
```

Comment:
It would be good to include some small introductory paragraph here (as this is the most user-facing document) similar to these ones written by @rhettinger:

https://docs.python.org/3/whatsnew/3.8.html#new-features

Reply:
It's also not really a "builtin type". I'd say something like "new type operators to allow writing `X | Y` instead of" (note that `X | Y` should be marked up as code).

Reply:
Thanks, I was under the impression that 'what's new' documents could have short entries for now since the 'Rules for maintenance' states that "It's OK to just add a fragmentary note about a change (...) The maintainer will research the change and write the necessary text.". However I realised that it would probably save whoever's editing it near release more time, so I'll try writing something more user-friendly.